### PR TITLE
Moving Astropy Compilation Team Docs onto GitHub

### PR DESCRIPTION
**Pull request by ceb8**
This PR moves [Compilation of the Team Docs of the Astropy Project](https://docs.google.com/document/d/19Hj8v4yP2q5Hb5G4KMLdDcjf6yQEH_9z104d0O6C34g/edit) (permission required) into GitHub.

### CODE REVIEW

File: policies/README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+### Coordination Committee
+- Policies: [CoCo Operating Policies](coco-operating-policies.md)
+- Records: [Running Notes](https://docs.google.com/document/d/1t7gEE6gIgNUCyObQFq3aYFMcOh4V4m27CNrxBGYwQuw/edit)
+- Internal Communication
+  - Bi-weekly closed meetings on Project calendar
+  - Private slack channel
+- External Communication
+  - Email: coordinators@astropy.org
+  - Github team [@astropy/coordinators](https://github.com/orgs/astropy/teams/coordinators)
+  - Quarterly updates to email lists.
```
Comment (pllim) [inline policies/README.md:16]: I didn't know we write quarterly updates. Is that still a thing?

But I think we do present a "State of Astropy" at Astropy Coordination Meetings.

File: policies/README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+  - Quarterly-ish updates to email lists.
+
+### Learn Team
+- Policies: No formal policies.
+- Records: Archival [Running Notes](https://docs.google.com/document/d/1T6FL0_XhnMEtI1Qq_KNZZzD0HNuVR3wHhhDYj9d21z0/edit)
+- Communication (all open): #learn channel on Astropy Slack
+
+### Infrastructure/DevOps Team
+- Policies: No formal policies. 
+- Records: [Running Notes](https://docs.google.com/document/d/15JSFh3OMF9Iz6ov3q_xxGO_BL8hRnuse4IMUrqEIvcg/edit)
```
Comment (pllim) [inline policies/README.md:39]:
```suggestion
- Records: Archival [Running Notes](https://docs.google.com/document/d/15JSFh3OMF9Iz6ov3q_xxGO_BL8hRnuse4IMUrqEIvcg/edit)
```

File: policies/README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+- Communication (all open):
+  - Regular open meetings on Project calendar
```
Comment (pllim) [inline policies/README.md:41]: That died.
```suggestion
```

File: policies/README.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+  - #infrastructure Slack channel
+  - Github team: [@astropy/devops-maintainers](https://github.com/orgs/astropy/teams/devops-maintainers) 
+    (this group only contains a subset of those who attend the tag-ups)
+  - Open issues/PRs on relevant Astropy org GitHub repos and tag @astropy/devops-maintainers.
+
+
```
Comment (pllim) [inline policies/README.md:47]:
```suggestion
```

**Review by pllim — COMMENTED** [review]
Do we want to break off the email `@` just to make spam bot's life a little harder?

Thanks!

File: policies/README.md (hunk 1 above)
Comment (ceb8) [reply policies/README.md:16]: Ha yes, I wasn't sure about this either. I am guessing it's not?

**Comment by ceb8** [comment]
Per discussion in CoCo meeting and having implemented the suggestions, I am merging this.

**Pull request merged**